app/services/submission_service.rb

```diff
@@ -101,6 +103,12 @@ def update_submission(
       submission.save!
     end
 
+    def notify_user_with_braze(submission_id, response)
+      if response
+        SwaMyCollectionEmailWorker.perform_in(1.day.from_now, submission_id)
+      end
+    end
+
```

Comment:
This is here only because Rubocop was raising that there are 2 levels of conditions on that method. We need a separate ticket to refactor this whole class.

Reply:
Here is an empty ticket if you'd like to add any details to guide the refactoring:

https://artsyproduct.atlassian.net/browse/SWA-265